pyproject.toml

```diff
@@ -8,6 +8,73 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "Cartopy"
+authors = [
+    {name = "UK Met Office"},
+]
+description = "A Python library for cartographic visualizations with Matplotlib"
+readme = "README.md"
+requires-python = ">=3.9"
+keywords = ["cartography", "map", "transform", "projection", "pyproj", "shapely", "shapefile"]
+license = {file = "COPYING.LESSER"}
+classifiers = [
+    'Development Status :: 4 - Beta',
+    'Framework :: Matplotlib',
+    'License :: OSI Approved :: GNU Lesser General Public License v3 or later (LGPLv3+)',
+    'Operating System :: MacOS :: MacOS X',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: POSIX :: AIX',
+    'Operating System :: POSIX :: Linux',
+    'Programming Language :: C++',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3 :: Only',
+    'Topic :: Scientific/Engineering',
+    'Topic :: Scientific/Engineering :: GIS',
+    'Topic :: Scientific/Engineering :: Visualization',
+]
+dependencies = [
+    "numpy>=1.21",
+    "matplotlib>=3.4",
+    "shapely>=1.7",
+    "packaging>=20",
+    "pyshp>=2.1",
+    "pyproj>=3.1.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+doc = ["beautifulsoup4", "pydata-sphinx-theme", "sphinx", "sphinx-gallery"]
+extras = ["pykdtree", "fiona"]
```

Comment:
I'm not sure `extras` is a useful name for the feature.

Reply:
Yeah, I was just copying over from the previous `requirements` directory files. I think we could re-evaluate which of these categories we want. I renamed it to `speedups` in a separate commit/branch.